particula/tests/unitless_test.py

```diff
@@ -1,24 +1,22 @@
-""" A quick test of strip_units.unitless utility
```

Comment:
Nit: rename the file to units_test to be consistent with the source file?

Reply:
Yes, good catch!